phys2bids/phys2bids.py

```diff
@@ -203,6 +207,31 @@ def _main(argv=None):
     outfile = os.path.join(options.outdir,
                            os.path.splitext(os.path.basename(options.filename))[0])
 
+    # Create logfile name
+    basename = 'phys2bids_'
+    extension = 'tsv'
+    isotime = datetime.datetime.now().replace(microsecond=0).isoformat()
```

Comment:
Can we add the version number in the filename *and* as the first log info after execution?

Reply:
I don't think the version number should go into the filename, that would make it unreadable. I'd say that the first log being the version number should be enough.

Reply:
Yeah, you're right.
However, I'm thinking about parallel computation - can we add the name of the input in the logger, or check that the same logger doesn't exists, and if so give it a different name?

Reply:
This is an example of the log filename: `phys2bids_2020-02-11T19:46:08.tsv`. 

I'll add the input to the log file, it sounds like a good idea. However, I don't expect two log files to have the same name. See how the name even contains the second in which the command was executed. I don't think this will happen but if it did, I don't think they would be overwritten.

Reply:
Alternatively, can we specify a destination for it? That way who wants to parallelise can save them in two different places.